pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -97,21 +99,27 @@ type ActualStateOfWorld interface {
 	// VolumeNodeExists returns true if the specified volume/node combo exists
 	// in the underlying store indicating the specified volume is attached to
 	// the specified node.
-	VolumeNodeExists(volumeName v1.UniqueVolumeName, nodeName types.NodeName) bool
+	IsVolumeAttachedToNode(volumeName v1.UniqueVolumeName, nodeName types.NodeName) bool
 
 	// GetAttachedVolumes generates and returns a list of volumes/node pairs
-	// reflecting which volumes are attached to which nodes based on the
+	// reflecting which volumes might attached to which nodes based on the
```

Comment:
worth adding more commentary on why it only "might" be attached, what case will it not be?

Reply:
fixed

Reply:
because we will keep records of all volumes even if the attach returns errors, the volume could be attached or not attached to the node.